joebvp/VPmeasure.py

```diff
@@ -272,6 +272,14 @@ def __init__(self,specfilename,parfilename=None,wave1=None,wave2=None,numchunks=
         cfg.normflux=self.normflux
         cfg.filename=self.specfilename
 
+        # define bad pixels
+        cond_badpix = (self.spectrum.wavelength <= self.spectrum.wvmin) | \
+                      (self.spectrum.wavelength >= self.spectrum.wvmax) | \
+                      (self.spectrum.sig <= 0) | \
+                      (self.spectrum.flux / self.spectrum.sig < cfg.min_sn)  # bad S/N
```

Comment:
Fair enough, that's why I left it as a free parameter in the configuration file. We could adopt `cfg.min_sn = -np.inf` as default if that's a concern (or simply remove it). I was only attempting at getting rid of these spectrum gaps somehow, but it will certainly exclude good quality pixels if these are genuinely black.